internal/kokoro/presubmit.sh

```diff
@@ -65,6 +65,12 @@ runPresubmitTests() {
   if [ -f "emulator_test.sh" ]; then
     ./emulator_test.sh
   fi
+
+  # Run conformance tests against a test proxy.
+  if [ -f "conformance_test.sh" ]; then
+    ./conformance_test.sh
+  fi
```

Comment:
I'm a bit confused about how exit codes are propagated here. `continuous.sh` seems to add together the exit_code after each sub-test, but presubmit seems to work differently.

Are you supposed to add ` | tee sponge_log.log` here so the error can be caught by `go-junit-report` in the line below? Or am I misunderstanding this section?

Reply:
The call to conformance_test.sh populates sponge.log if I'm reading this correctly - teeing here would duplicate output.

Reply:
Right